src/encoded/tests/test_permissions.py

```diff
@@ -515,11 +514,56 @@ def test_submitter_can_view_ownitem(ind_human_item, submitter_testapp, wrangler_
         submitter_testapp.get(res.json['@graph'][0]['@id'], status=200)
 
 
-def test_submitter_cannot_view_ownitem_replaced(ind_human_item, submitter_testapp,
-                                                wrangler_testapp):
+def test_submitter_cannot_view_ownitem_replaced_using_accession(ind_human_item, submitter_testapp, wrangler_testapp):
+    res = submitter_testapp.post_json('/individual_human', ind_human_item, status=201)
+    wrangler_testapp.patch_json(res.json['@graph'][0]['@id'], {"status": "replaced"}, status=200)
+    my_accession = '/' + res.json['@graph'][0]['accession']
+    submitter_testapp.get(my_accession, status=404)
+
+
+def test_submitter_can_view_ownitem_replaced_using_uuid(ind_human_item, submitter_testapp, wrangler_testapp):
     res = submitter_testapp.post_json('/individual_human', ind_human_item, status=201)
     wrangler_testapp.patch_json(res.json['@graph'][0]['@id'], {"status": "replaced"}, status=200)
-    submitter_testapp.get(res.json['@graph'][0]['@id'], status=404)
+    my_uuid = '/' + res.json['@graph'][0]['uuid']
+    submitter_testapp.get(my_uuid, status=301)
```

Comment:
The 301 is a 'moved permanently' response ah I see so it redirects to the new identifier/url.  Maybe it's worth testing that the url is indeed referring to the same item?

Maybe this is the best we can do in unit testing for 301 redirects but when you get a 301 you really aren't getting the item.